amplify/backend/function/questionslist/src/questions.js

```diff
@@ -0,0 +1,65 @@
+const AWS = require('aws-sdk');
```

Comment:
This is less of a request for change but more of food for thought.

What do you think about using import/export here instead of require()? [Import](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#specifications)/[export](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export#specifications) is part of the ES6 standard and require() is not so I was thinking maybe we should use that instead since ES6 standard will only be more and more supported in the future. But honestly since this works I think it's fine to stick with this for now too.

Reply:
Agreed, I think it's good to stay consistent with everything else within the function and React code since most of it is also ES6

Reply:
Actually after looking into it, there needs to be quite a bit more setup to have ES6 imports work since Lambda by default uses CommonJS to import our `handler` function and throws errors when ES6 module imports are used, so something like babel/webpack is required (which I don't think is necessary for this tiny function)

Reply:
Sounds good to me